src/client/gl/gl_errors.cpp

```diff
@@ -3,6 +3,93 @@
 #include <glad/glad.h>
 #include <iostream>
 
+void gl_debug_callback(GLenum source, GLenum type, GLuint id, GLenum severity,
```

Comment:
(Just to fit in with the rest of the code style)

```suggestion
void glDebugCallback(GLenum source, GLenum type, GLuint id, GLenum severity,
```